robottelo/cli/sm_service.py

```diff
@@ -27,50 +27,50 @@ class Service(Base):
     command_base = 'service'
 
     @classmethod
-    def start(cls, options=None):
+    def start(cls, options=None, env_var=''):
```

Comment:
```suggestion
    def start(cls, options=None, env_var=None):
```
same as comment - https://github.com/SatelliteQE/robottelo/pull/10227#discussion_r1022466973

Reply:
Updated